can-bus/json/PDM/PDM_tx.json

```diff
@@ -12,17 +12,34 @@
             }
         }
     },
-    "PDM_Errors": {
-        "msg_id": 403, 
-        "cycle_time": 1000,
-        "description": "Errors on the PDM.",
+    "PDM_CanFifoOverflow": {
+        "msg_id": 403,
+        "cycle_time": 5000,
+        "description": "PDM tx/rx fifo overflow count",
         "signals": {
             "TxOverflowCount": {
-                "bits": 8
+                "bits": 32
             },
             "RxOverflowCount": {
-                "bits": 8
-            },
+                "bits": 32
+            }
+        }
+    },
+    "PDM_WatchdogTimeout": {
+        "msg_id": 404,
+        "cycle_time": 1000,
+        "description": "PDM Watchdog Timeout Signal",
+        "signals": {
+            "TaskName": {
+                "enum": "RtosTaskName"
+            }
+        }
+    },
+    "PDM_Warnings": {
```

Comment:
re-organized messages to be more in line with BMS

Reply:
Hmm any reason you separated the warnings, watchdog timeout, and fifo overflow into seperate messages? Its more efficient on bus load to have bigger messages

Reply:
Nothing really, I just was following the same format I followed for the BMS, to have the warnings message containing only warning flags, but if this is more efficient we should do it that way. I feel we'll likely have to revisit the json's for each board during cleanup to make sure each board follows the same message format